app.js

```diff
@@ -168,6 +173,13 @@ d.run(function () {
 			var cors = require('cors');
 			var app = express();
 
+			if (appConfig.coverage) {
+				var im = require('istanbul-middleware');
+				logger.debug('Hook loader for coverage - ensure this is not production!');
```

Comment:
`ensure this is not production!` > `do not use in production environment!`

What are actual implications here?